api/Window.json

```diff
@@ -5709,6 +5709,153 @@
           }
         }
       },
+      "URL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "41"
+              },
+              {
+                "version_added": "8",
+                "version_removed": "41",
+                "notes": [
+                  "Implemented under the non-standard name <code>webkitURL</code>."
+                ]
+              }
+            ],
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "19",
+                "notes": [
+                  "From Gecko 2 (Firefox 4) to Gecko 18 included, Gecko returned an object with the non-standard <code>nsIDOMMozURLProperty</code> internal type. In practice, this didn't make any difference."
+                ]
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "14",
+                "version_removed": "19",
+                "notes": [
+                  "From Gecko 2 (Firefox 4) to Gecko 18 included, Gecko returned an object with the non-standard <code>nsIDOMMozURLProperty</code> internal type. In practice, this didn't make any difference."
+                ]
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "15",
+              "notes": [
+                "Implemented under the non-standard name <code>webkitURL</code>."
+              ]
+            },
+            "opera_android": {
+              "version_added": "15",
+              "notes": [
+                "Implemented under the non-standard name <code>webkitURL</code>."
+              ]
+            },
+            "safari": [
+              {
+                "version_added": "7"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "7",
```

Comment:
Tested Safari myself and it doesn't look like the prefix has been removed yet

Reply:
Ok. This I just used based on the info in the old table. I'll revert it.